data/json/items/gun/308.json

```diff
@@ -76,6 +76,7 @@
     "reload": 400,
     "modes": [ [ "DEFAULT", "auto", 100 ] ],
     "magazines": [ [ "308", [ "belt308" ] ] ],
+    "//": "Its barrels are rotated by an electric motor, powered by UPS. After the UPS system is fixed, this should use electrical power."
```

Comment:
As a response to your previous comment months ago, I did read your item_group descriptions when assigning guns to item_groups. They seemed to say that the item_groups were looser than your later comment would indicate, so I put the guns where theey made sense.